files/bootstrap.sh

```diff
@@ -392,6 +399,22 @@ Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
 EOF
 fi
 
+# Replace dockerd with containerd
+if [[ "$CONTAINER_RUNTIME" -eq "containerd" ]]; then
+    cat <<EOF > /etc/systemd/system/kubelet.service.d/50-kubelet-container-runtime.conf
```

Comment:
We need to quote the `EOF` like `cat <<'EOF'` else it will try to expand the variable `KUBELET_ARGS` and fail because the variable is not defined. See man bash [1]:

> If any characters in word are quoted, the delimiter is the result of quote removal on word, and the lines in the here-document are not expanded. If word is unquoted, all lines of the here-document are subjected to parameter expansion, command substitution, and arithmetic expansion. 

[1] https://linux.die.net/man/1/bash